src/shared/libfido2-util.c

```diff
@@ -1124,7 +1124,7 @@ int fido2_list_devices(void) {
                 goto finish;
         }
 
-        t = table_new("path", "manufacturer", "product");
+        t = table_new("path", "manufacturer", "product", "hmac secret");
```

Comment:
maybe instead of "hmac secret" say "compatible"? or "supported" or so?

otherwise on first look without consulting the man page it might suggest that there's already an hmac secret enrolled with this or so. And I think "compatible" or "supported" is a lot more explanatory, since it doesn't bother about technical details

Reply:
Updated to say "compatible".